headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/resource/v1_0/WarehouseItemResourceImpl.java

```diff
@@ -104,6 +106,43 @@ public WarehouseItem getWarehouseItem(Long id) throws Exception {
 				GetterUtil.getLong(id)));
 	}
 
+	@Override
+	public Page<WarehouseItem> getWarehouseItemsUpdatedPage(
+			Date start, Date end, Pagination pagination)
+		throws Exception {
+
+		if ((start != null) && (end != null) && (start.compareTo(end) > 0)) {
+			throw new IllegalArgumentException(
+				"End date should be after start date");
+		}
+
+		if ((start == null) && (end == null)) {
+			throw new IllegalArgumentException(
+				"At least one date should be defined");
+		}
+
+		if (start == null) {
+			start = DateUtils.addDays(end, -DateUtils.DEFAULT_INCREMENT_DAY);
```

Comment:
@andrea-ale-sbarra since you only use `DateUtils` here, can you just write `addDays` as a private method in this class, and not add the new class? Liferay tends to be pretty [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it), so we would generally only add a utility once we need it in multiple files. (Also, we tend to name things `*Util` rather than `*Utils`, for your future reference.)